tests/test_yaml.py

```diff
@@ -0,0 +1,93 @@
+import importlib
+import os
+from unittest import mock
+import pytest
+
+from rpmlb.yaml import Yaml
+import helper
+
+
+@pytest.fixture
+def valid_yaml():
+    """A Yaml object with a valid YAML file."""
+    return Yaml(helper.get_valid_recipe_file())
+
+
+def test_init_loads_file():
+    yaml = Yaml(helper.get_valid_recipe_file())
+    assert yaml
+
+
+def test_init_load_default_loader_when_failing_to_import_c_loader():
```

Comment:
That's exactly why I don't like coverage hunting. You are chasing a ghost. No test for this is needed, unless you are trying to test if Python works.